ensembl_dbcopy/models.py

```diff
@@ -171,5 +174,7 @@ class Meta:
         verbose_name = 'Host Group'
 
     group_id = models.BigAutoField(primary_key=True)
-    host_id = models.ForeignKey(Host, db_column='auto_id', on_delete=models.CASCADE, related_name='groups')
+    #host_id = models.ForeignKey(Host, db_column='auto_id', on_delete=models.CASCADE, related_name='groups')
```

Comment:
You shouldn't be using the same group table. going to be a mess in the long run. This initial group model is supposedly dedicated to "Users's group" and determine the user's group permissions on a host.

Reply:
should i creatge new table named 'hostTogroup'  and this should be in sync with auth_group table in production_service db